nixos/modules/system/boot/systemd/ultrablue.nix

```diff
@@ -0,0 +1,39 @@
+{ pkgs, config, lib, ... }:
+
+let
+  cfg = config.boot.initrd.systemd.ultrablue;
+in
+{
+  options.boot.initrd.systemd.ultrablue = {
+    enable = lib.mkEnableOption (lib.mdDoc "the Ultrablue remote attestation server");
+    package = (lib.mkPackageOptionMD pkgs "Ultrablue server package" {
+      default = "ultrablue-server";
+    });
+  };
+  config = lib.mkIf cfg.enable {
+    boot.initrd.systemd = {
+      storePaths = [
+        "${cfg.package}/bin/ultrablue-server"
+        "${pkgs.toybox}/bin/sleep"
+      ];
+
+      services.ultrablue-server = {
+        after = [ "bluetooth.service" ];
+        wants = [ "bluetooth.service" "cryptsetup-pre.target" ];
+        before = [ "cryptsetup.target" ];
+        wantedBy = [ "cryptsetup.target" ];
+
+        unitConfig.DefaultDependencies = false;
+
+        serviceConfig = {
+          Type = "oneshot";
+          # TODO: upstream hack, brrr
+          ExecStartPre = "${pkgs.toybox}/bin/sleep 5";
+          ExecStart = "${cfg.package}/bin/ultrablue-server";
+          TimeoutSec = 60;
+          StandardOutput = "tty";
```

Comment:
I know this is all copied from [the upstream file](https://github.com/ANSSI-FR/ultrablue/blob/dev/server/unit/ultrablue-server.service), but I'd like to avoid repeating the same mistakes.

1. `oneshot` doesn't seem right to me.
2. Can we order this after an appropriate unit rather than just sleeping for 5 seconds?
3. I can't imagine `StandardOutput = "tty"` is correct, is it?

Reply:
It was already ordered after an appropriate unit, so the question is: what do we need to wait on? This is yet to be determined for me. :)